pages/common/laravel.md

```diff
@@ -0,0 +1,15 @@
+# laravel
+
+> The PHP Framework For Web Artisans.
+
+- Create a new Laravel application:
+
+`new`
+
+- List the available commands:
+
+`list`
+
+- Display help for a command:
+
+`help`
```

Comment:
From this example I can't tell how to specify the command I need help with. Perhaps we could expand on this?